CPAC/registration/tests/test_ants_registration_connector.py

```diff
@@ -0,0 +1,45 @@
+import pytest
+from CPAC.registration.registration import ANTs_registration_connector
+
+
+class AttrDict(dict):
+    def __getattr__(self, item):
+        value = self[item]
+        if isinstance(value, dict):
+            return AttrDict(value)
+        return value
+
+
+@pytest.mark.parametrize("sink_native_transforms", [True, False])
+def test_ants_registration_connector(sink_native_transforms):
+    wf_name = "test_ants_registration_connector"
+    cfg = AttrDict(
+        {
+            "registration-workflows": {
+                "sink_native_transforms": sink_native_transforms
+            },
+            "pipeline_setup": {"system_config": {"num_ants_threads": 1}},
+            "registration_workflows": {
+                "sink_native_transforms": sink_native_transforms,
+                "anatomical_registration": {
+                    "reg_with_skull": True,
+                    "registration": {"ANTs": {"use_lesion_mask": False}},
+                },
+            },
+        }
+    )
```

Comment:
(if we import `Configuration` instead of creating `AttrDict`)
```suggestion
    cfg = Configuration(
        {
            "pipeline_setup": {"system_config": {"num_ants_threads": 1}},
            "registration_workflows": {
                "sink_native_transforms": sink_native_transforms,
                "anatomical_registration": {
                    "reg_with_skull": True,
                    "registration": {"ANTs": {"use_lesion_mask": False}},
                },
            },
        }
    )
```